docker_reverse_proxy/Dockerfile

```diff
@@ -0,0 +1,14 @@
+FROM nginx
+
+# cleanup unused nginx config
+RUN rm /etc/nginx/conf.d/default.conf
+
+# copy our own nginx config
+COPY ./conf/reverse-proxy.conf.template /etc/nginx/conf.d
+COPY ./conf/nginx.conf /etc/nginx/nginx.conf
+
+# copy startup script
+COPY ./docker-entrypoint.sh /docker-entrypoint.sh
+RUN chmod +x /docker-entrypoint.sh
+
+ENTRYPOINT ["bin/bash", "/docker-entrypoint.sh"]
```

Comment:
The leading slash is missing, it's better to specify the full path (`/bin/bash`).